client-app/ui-kit/components/atoms/input/vc-input.vue

```diff
@@ -50,6 +53,7 @@ const props = defineProps({
   isReadonly: Boolean,
   isDisabled: Boolean,
   isRequired: Boolean,
+  isBordered: { type: Boolean, default: true },
```

Comment:
Maybe `noBorder: Boolean` or `withoutBorder: Boolean`

To make it so:
  `<VcInput no-border />` or `<VcInput without-border />`